docs/API/V1/contents.md

```diff
@@ -88,11 +91,44 @@ Note that symlinks within the dataset representation will result in a URI return
 }
 ```
 
-### File
+#### Subirectory object
+
+The subdirectory object gives the name of the directory, the type of the entry,
+and a URI that can be used with a subsequent `GET` operation to return a
+[directory object](#directory-object) for that nested path.
 
-When the specified path is a leaf file (regular file or symlink), the response body is an `application/json` formatted [file object](#file-object), including the name and filesystem metadata. A URI is provided which can be used to `GET` the file's contents as a raw byte stream.
+When a directory contains a symlink to a directory, that subdirectory name will
+appear in the `directories` list, but will be designated with a `type` of
+`sym` instead of `dir`.
 
-#### Example
+The `type` codes are:
+* `dir`: Directory
+* `sym`: Symbolic link
+
+```json
+{
+    "name": "reference-result",
+    "type": "sym",
+    "uri": "http://hostname/api/v1/datasets/contents/<id>/sample1"
+}
+```
+
+#### File object
```

Comment:
Should this have three `#`'s to match line 52 ("Directory Object")?  (Lines 49 & 50 reference that and this, respectively, so they seem like they should be peers (although, I can see how you want this to match line 94 ("Sub[d]irectory Object").)